api-gateway/pom.xml

```diff
@@ -36,11 +36,40 @@
             <groupId>org.springframework.cloud</groupId>
             <artifactId>spring-cloud-starter-gateway</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-security</artifactId>
+        </dependency>
```

Comment:
Spring Security isn't used in the API gateway for now. Let's remove the dependency and security configurations.